src/pages/OfferNew/Checkout/index.tsx

```diff
@@ -274,6 +275,7 @@ export const Checkout: React.FC<Props> = ({
             signQuotesMutation.loading ||
             signUiState === SignUiState.STARTED ||
             signUiState === SignUiState.STARTED_WITH_REDIRECT ||
+            !hasContract ||
```

Comment:
won't this cause the loading state to always be true until you signed?

Reply:
Really good catch! Fixed now